cves/CVE-2009-2693.yml

```diff
@@ -1,205 +1,216 @@
+
+bounty_instructions: |
+  If you came across any indications that a bounty was paid out for this
+  vulnerability, fill it out here. Or correct it if the information already here
+  was wrong. Otherwise, leave it blank.
+bounty:
+  amt:
+  announced: 
+  url:
+bugs: []
+fixes_vcc_instructions: |
+  Please put the Git commit SHA in "commit" below, and any notes about how this
+  was discovered in the "note" field.
+  Refer to our instructions on how to find a Git SHA from an SVN revision.
+fixes:
+   - commit: 3e1010b1a2f648581fac3d68afbf18f2979f6bf6
+     note: Fixed Dec 21 2009 at 12:25:14
+vccs:
+  - commit: eae54419c6e196933998f63358367040edaa4a8c
+    note: Pushed Jul 20 2009 at 12:50:34
+
+incomplete_fix_instructions: |
+  Did the above "fixes" actually fix the vulnerability?
+  Please list any fix commits for this vulnerability that had to be corrected
+  at a later date.
+incomplete_fixes:
+  - commit: none
+    note: The fix worked
+
+upvotes_instructions: |
+  Students: when initially writing this, ignore this upvotes number.
+  Once this work is being reviewed, you will be giving a certain amount of
+  upvotes to each vulnerability you see. Your peers will tell you how
+  interesting they think this vulnerability is, and you'll add that to the
+  upvotes score on your branch.
+upvotes: 2
+
+unit_tested:
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+    Write the reasoning behind your answer in the "answer" field.
+    For the "code" answer below, look not only at the fix but the surrounding
+    code near the fix and determine if and was there were unit tests involved
+    for this module. Must be just "true" or "false".
+    For the "fix" answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+    Must be just "true" or "false".
+  answer: false
+  code: false
+  fix: 
+
+discovered:
+  question: |
+    How was this vulnerability discovered?
+    Go to the bug report and read the conversation to find out how this was
+    originally found.
+    * Answer in longform below in "answer"
+    * Fill in the date in YYYY-MM-DD
+    * If it's clear that the vulnerability was discovered by a contest,
+      fill in the name there.
+    * The "automated" flag can be true, false, or nil.
+    If there is no evidence as to how this vulnerability was found, then you
+    may leave the entries blank except for "answer", BUT please write down
+    where you looked in "answer".
+  answer: The SVN page for the commit does not indicate how the vulnerability was found
+    https://svn.apache.org/repos/asf/tomcat/trunk@892795
+    and the links within the CVE page also do not show how this was discovered
+    https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2009-2693
+  date:
+  automated:

```

Comment:
Automated field is blank?

Reply:
I guess I forgot to fill this one in. Thanks for letting me know!